tox.ini

```
@@ -2,8 +2,19 @@
 envlist = py,typing,lint,docs,coverage-report
```

```suggestion
envlist = docker,typing,lint,docs,coverage-report
```

The specific `docker` environment must be run instead of the default environment (essentially `py`) because that's the one where the docker containers are running.

---

Also note the list of environments will need to be similarly updated in the GitHub Actions workflow.